src/library/Asset.ts

```diff
@@ -37,15 +37,17 @@ export class Asset {
     return await this.methods.getPositions();
   };
 
-  // TODO getPositionCR
-  // getPositionCR = async () => {
-  //   return await this.methods.getPositionCR(this.asset);
-  // };
-
-  // TODO getGCR
-  // getGCR = async () => {
-  //   return await this.methods.getGCR(this.asset);
-  // };
+  getPositionCR = async () => {
+    return await this.methods.getPositionCR(this.asset);
+  };
+
+  getUserStats = async () => {
+    return await this.methods.getUserStats(0, 1621298935)
```

Comment:
any reason this is passed here? `1621298935` can just be handled inside the function itself